scripts/bat/ofs/ofs_bat.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+#
+# IOFS EA Test cases
+
+echo "------IOFS EA Test cases--------"
+
+
+ opae_unit_tests()
+ {
+ echo "--opae_unit_tests--"
+ rm -rf opae-sdk
+ git clone https://github.com/OPAE/opae-sdk.git
+ cd opae-sdk
+ mkdir mybuild
+ cd mybuild
+  echo "--START HW UNIT TESTS--"
+ cmake ..   -DCMAKE_BUILD_TYPE=Debug -DOPAE_BUILD_TESTS=ON -DOPAE_ENABLE_MOCK=OFF
+ make -j
+ OPAE_EXPLICIT_INITIALIZE=1 LD_LIBRARY_PATH=$PWD/lib make test
+ echo "--END HW UNIT TESTS--"
+
+ echo "--START CREATE RPM--"
+ rm -rf *
+ cmake ..   -DCPACK_GENERATOR=RPM -DOPAE_BUILD_LEGACY=ON -DOPAE_BUILD_SIM=ON
+ make -j
+ make package_rpm
+ yum -y remove opae*
+ yum -y install opae-*
+ cd ../..
+ echo "--END CREATE RPM--"
+ }
+  #opae_unit_tests
+
+
+found=$(ls /dev/dfl-* | wc -l)
+echo $found
+if [[ $found -lt 1 ]]; then
+    echo "Failed to locate drivers."
+    exit 1
+fi
+
+found=$(lspci | grep 0b2b)
+echo $found
+if [ "$1" != "" ]; then
+  bus_num=$1
+  if [[ $found != *"$bus_num"* ]]; then
+      echo "Invalid bus number param: "$bus_num
+      exit 1
+  fi
+else
+  bus_num=${found:0:2}
+  if [[ $((16#$bus_num)) -lt 0 ]]; then
+      echo "Invalid bus number: "$bus_num
+      exit 1
+  fi
+fi
+
+gbs_file=nlb_mode_0_unsigned.gbs
+iofs_file=ofs_fim_page1_unsigned.bin
```

Comment:
Is it safe to assume that these files will exist in the current working directory?